coilsnake/model/eb/blocks.py

```diff
@@ -12,4 +18,78 @@ def from_compressed_block(self, block, offset=0):
 
     def compress(self):
         compressed_data = comp(self.to_list())
-        self.from_list(compressed_data)
\ No newline at end of file
+        self.from_list(compressed_data)
+
+
+class EbRom(Rom):
+
+    # The MD5 hash for the reference EB ROM (clean, unheadered, unexpanded).
+    REFERENCE_MD5 = "a864b2e5c141d2dec1c4cbed75a42a85"
+
+    # The MD5 hashes for the alternative EB ROMs and the associated patches for
+    # turning them into the reference ROMs.
+    ALT_MD5 = {
+        "8c28ce81c7d359cf9ccaa00d41f8ad33": "fix1.ips",
+        "b2dcafd3252cc4697bf4b89ea3358cd5": "fix2.ips",
+        "0b8c04fc0182e380ff0e3fe8fdd3b183": "fix3.ips",
+        "2225f8a979296b7dcccdda17b6a4f575": "fix4.ips",
+        "eb83b9b6ea5692cefe06e54ea3ec9394": "fix5.ips",
+        "cc9fa297e7bf9af21f7f179e657f1aa1": "fix6.ips"
+    }
+
+    def add_header(self):
```

Comment:
Ah, I get it, it's because the parent method can raise NotImplementedError, so PyCharm thinks it's an abstract method. I'll remove this one and the one below.